src/migrations/20240405174629-jobs.js

```diff
@@ -0,0 +1,26 @@
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+        CREATE TABLE IF NOT EXISTS jobs (
+            id SERIAL PRIMARY KEY NOT NULL,
+            name VARCHAR(255) NOT NULL,
```

Comment:
```suggestion
            name TEXT NOT NULL,
```
VARCHAR(255) does not have value anymore since postgres ~10 and with postgres 14 indexing is better with TEXT